src/OpenTelemetry.Collector.Dependencies/Implementation/HttpHandlerDiagnosticListener.cs

```diff
@@ -27,6 +28,8 @@ namespace OpenTelemetry.Collector.Dependencies.Implementation
 {
     internal class HttpHandlerDiagnosticListener : ListenerHandler
     {
+        private static readonly Regex CoreAppMajorVersionCheckRegex = new Regex("^\\.NETCoreApp,Version=v(\\d+)\\.", RegexOptions.Compiled);
```

Comment:
Maybe it would be good to make this case insensitive to help protect any any future capitalisation changes?

Reply:
That's why I was not a fan of predicting how future version format would look like :)

Reply:
Added `RegexOptions.IgnoreCase`.

For comparison, here's how AppInsights does it: 

https://github.com/microsoft/ApplicationInsights-dotnet/blob/d032dbfdab3059f894aaa0de139a01231ccb997c/WEB/Src/DependencyCollector/Shared/HttpCoreDiagnosticSourceListener.cs#L640

But I think that mechanism will also have an issue with 5.0?